### PR TITLE
[docs] cloud performance tab

### PR DESCRIPTION
**Pull request by ddhodge**
New Performance tab with metrics, live and slow queries

@netlify /latest/yugabyte-cloud/cloud-monitor/

The metrics table has gaps - more info needed on what the metrics mean to users and how they should potentially respond.

Questions on live/slow query tables:

- what are the limits on the number of queries in these tables? eg, max 1000, only from past 24hrs…etc
- how is a query determined to be slow? eg, fixed eg ‘1 second’, some dynamic threshold based on average time of operations, something else?

### CODE REVIEW

File: docs/content/latest/yugabyte-cloud/cloud-monitor/cloud-queries-live.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    identifier: cloud-queries-live
+    parent: cloud-monitor
+    weight: 200
+isTocNested: true
+showAsideToc: true
+---
+
+Evaluate the performance of running queries on your cluster using the **Live Queries** on the cluster **Performance** tab. You can use this data to:
+
+- Visually identify relevant database operations.
```
Comment (yb-andrew) [inline docs/content/latest/yugabyte-cloud/cloud-monitor/cloud-queries-live.md:18]: Should mention that live queries only show queries "in-flight" or currently in progress. This means queries that execute quickly might not show up on load of page.

**Review by yb-andrew — CHANGES_REQUESTED** [review]
The metric graphs should have exactly the same names as the platform metric graphs. Liza created a PR for Platform, you should be able to use the same information: https://github.com/yugabyte/yugabyte-db/pull/10471

There are no limits to the number of queries displayed in the tables. This can be a potential issue that we will address in the future.

Right now the "slow queries" show all the queries, not just the slow ones. So the name is a little misleading right now. We will be adding thresholds in a future release.

File: docs/content/latest/yugabyte-cloud/cloud-monitor/_index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    weight: 100
+---
+
+To ensure the cluster configuration matches its performance requirements, you can monitor key database performance metrics and [scale the cluster vertically or horizontally](../cloud-clusters/configure-clusters/) as your requirements change. 
+
+Yugabyte Cloud provides the following tools to monitor your clusters:
+
+- Performance metrics charted over time.
+- A list of live queries.
+- A list of slow running queries that have been run on the cluster.
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-monitor/_index.md:21]:
```suggestion
- A list of slow running YSQL queries that have been run on the cluster.
```

File: docs/content/latest/yugabyte-cloud/cloud-monitor/manage-clusters.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    weight: 500
+isTocNested: true
+showAsideToc: true
+---
+
+Review the status of the nodes in your cluster using the **Nodes** tab. The **Nodes** tab lists the nodes in the cluster, including the name, cloud, RAM, SST size, and read and write operations per second, and the node status.
+
+![Cloud Cluster Nodes tab](/images/yb-cloud/cloud-clusters-nodes.png)
+
+Yugabyte Cloud suppports horizontal scaling of clusters. If your workloads have increased, you can dynamically add nodes to a running cluster to improve latency, throughput, and memory. Likewise, if your cluster is over-scaled, you can reduce nodes to reduce costs. For information on scaling clusters, refer to [Configure clusters](../configure-clusters#infrastructure).
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-monitor/manage-clusters.md:20]: Configure clusters has a broken link.

**Review by yb-andrew — APPROVED** [review]
LG

File: docs/content/latest/yugabyte-cloud/cloud-monitor/overview.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+| YCQL Operations/Sec | The count of DELETE, INSERT, SELECT, and UPDATE transactions, as well as other statements through the YCQL API. | If the count drops significantly lower than your average count, this could indicate an application connection failure. |
+| YCQL Remote Operations | The number of remote read and write requests. Remote requests are re-routed internally to a different node for executing the operation. | If an application is using a Yugabyte driver that supports local query routing optimization and prepared statements, the expected value for this is close to zero. If using the Cassandra driver or not using prepared statements, expect to see a relatively even split between local and remoteoperations (for example, ~66% of requests to be remote for a 3-node cluster).
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-monitor/overview.md:44]:
```suggestion
| YCQL Remote Operations | The number of remote read and write requests. Remote requests are re-routed internally to a different node for executing the operation. | If an application is using a Yugabyte driver that supports local query routing optimization and prepared statements, the expected value for this is close to zero. If using the Cassandra driver or not using prepared statements, expect to see a relatively even split between local and remote operations (for example, ~66% of requests to be remote for a 3-node cluster). 
```

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Pull request merged**